src/connectedk8s/azext_connectedk8s/_client_factory.py

```diff
@@ -6,11 +6,9 @@
 from azure.cli.core.profiles import ResourceType
 from azure.cli.core._profile import Profile
 from azure.cli.core.commands.client_factory import configure_common_settings
-from azure.cli.core.commands.client_factory import get_subscription_id
```

Comment:
Can you once check if this import is not anymore used in that client_factory? If it is used, pls revert this.

Reply:
I did a control F, did not see it being used.

Reply:
Okay.